src/components/Bio/Bio.tsx

```diff
@@ -1,99 +1,116 @@
-  teamProjectDegree: number;
-  tel: string;
-  firstName: string;
-  lastName: string;
-  githubUsername: string;
-  portfolioUrls: string[];
-  projectUrls: string[];
-  bio: string;
-  expectedContractType: ContractType;
-  expectedSalary: number;
-  canTakeApprenticeship: choiceYesNO;
-  monthsOfCommercialExp: number;
-  education: string;
-  workExperience: string;
+    id: string;
+    email: string;
+    courseCompletion: number;
+    courseEngagment: number;
+    projectDegree: number;
+    teamProjectDegree: number;
+    tel: string;
+    firstName: string;
+    lastName: string;
+    githubUsername: string;
+    portfolioUrls: string[];
+    projectUrls: string[];
+    bio: string;
+    expectedContractType: ContractType;
+    expectedSalary: number;
+    canTakeApprenticeship: choiceYesNO;
+    monthsOfCommercialExp: number;
+    education: string;
+    workExperience: string;
 };
 
 export const Bio = (props: Props) => {
-  const [gitHubAccountTrue, setGitHubAccountTrue] = useState(false);
+    const navigate = useNavigate()
+    const [gitHubAccountTrue, setGitHubAccountTrue] = useState(false);
+    const [removeFromToTalk] = useRemoveFromToTalkMutation()
+    const [studentHired] = useStudentHiredMutation()
+    const handleRemoveFromToTalkList = async () => {
+        await removeFromToTalk(props.id)
+        navigate('/hr')
+    }
+   const handleStudentStatusChange = async () => {
+        await studentHired(props.id);
+        navigate('/hr')
+    }
+    const handleReturnToMainPage = ()=>{
+        navigate('/hr')
+    }
+    useEffect(() => {
+        (async () => {
+            const checkGitHub = await checkGitHubAccount(props.githubUsername);
+            checkGitHub === 200 && setGitHubAccountTrue(true);
+        })();
+    }, []);
 
-  useEffect(() => {
-    (async () => {
-      const checkGitHub = await checkGitHubAccount(props.githubUsername);
-      checkGitHub === 200 && setGitHubAccountTrue(true);
-    })();
-  }, []);
+    return (
+        <div className='bio'>
+            <div className='bio__img'>
+                <img
+                    src={
+                        gitHubAccountTrue
+                            ? `https://github.com/${props.githubUsername}.png`
+                            : require('../../utils/img/github.png')
+                    }
+                    alt='avatar'
+                />
+            </div>
 
-  return (
-    <div className='bio'>
-      <div className='bio__img'>
-        <img
-          src={
-            gitHubAccountTrue
-              ? `https://github.com/${props.githubUsername}.png`
-              : require('../../utils/img/github.png')
-          }
-          alt='avatar'
-        />
-      </div>
+            <div className='bio__details'>
+                <p className='bio__name'>{`${props.firstName} ${props.lastName}`}</p>
+                <a
+                    href={`https://github.com/${gitHubAccountTrue ? props.githubUsername : null}`}
```

Comment:
odrazu tutaj jak zrobimy destrukturyzację to można tam przypisać default value na null i tutaj ominiemy taki warunkowy zapis

Reply:
default value zamiast null dałem pusty string, bo inaczej TS marudzi